ci/gather_diagnostics_data.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status, or zero if all
+# commands in the pipeline exit successfully.
+set -eu
+
+readonly SCRIPT_DIRECTORY="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+export TTY_OPTS="-t"
+
+if [ "${#}" -eq 0 ]; then
+  echo "The following arguments are required: services."
+  exit 1
+fi
+
+echo "Run data services diagnostic: $@"
```

Comment:
I'd get rid of this, it duplicates the checks and logging already in `ci_cli.py`.

Reply:
removed